adagrams/game.py

```diff
@@ -1,13 +1,107 @@
 from random import randint
 
 def draw_letters():
-    pass
+    LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    pile = []
+    hand = []
+    # created a list of letters
+    for k, v in LETTER_POOL.items():  
+        for counter in range(0, v):      
+            pile.append(k)
+
+    for i in range(10):
```

Comment:
Same comment as above, `i` is initialized, but not accessed so we should use `_` instead. 

Additionally, if I did not know how to play Adagrams, I would not understand the significance of 10. If I were a dev who tried to read this, I might be wondering "Why do we only loop 10 times"?

It would be more descriptive to use a constant variable to reference 10 so that your code is more self-documenting. 

```suggestion
    NUM_TILES_ALLOWED_IN_HAND = 10
    for _ in range(NUM_TILES_ALLOWED_IN_HAND):
```